tests/load/load_test.go

```diff
@@ -33,15 +33,27 @@ var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
 		blockchainID := subnetDetails.BlockchainID
 
 		nodeURIs := subnetDetails.ValidatorURIs
-		rpcEndpoints := make([]string, 0, len(nodeURIs))
+		nodeURIsLen := len(nodeURIs)
+		gomega.Expect(nodeURIsLen).Should(gomega.BeNumerically(">", 0))
```

Comment:
Was this an issue testing locally that `nodeURIs` ended up being 0?

Reply:
i can remove this now. i was accessing an element in a previous revision so i wanted to check if it had more than 0 values.